File: 5. Week/saturday 10.02/main.js
let obj = {
  1: 1,
  2: 2,
  3: 3,
};

let len = Object.keys(obj).length;
obj.g = "HHH";
let len2 = Object.keys(obj).length;
console.log(len2, "OBJ LEN");

File: 5. Week/saturday 10.02/main.py
# obj = {'a': 1, 'b':2, 'c':3}

# keys = obj.keys()
# obj['d'] = 4
# print(len(keys), 'LEN PY')

sent = 'This is a sample text. This text is a good example.'


def word_counter(sentence): 
    result = {}
    splitted_sent = sentence.split()
    for word in splitted_sent:
        word = word.strip('.')
        if result.get(word):
            result[word] += 1
        else:
            result[word] = 1
    
    print(result)


word_counter(sent)